pkg/path.bzl

```diff
@@ -61,8 +61,9 @@ def dest_path(f, strip_prefix, data_path_without_prefix = ""):
         return f_short_path[len(strip_prefix):]
     return f_short_path
 
-def compute_data_path(out, data_path):
+def compute_data_path(ctx, data_path):
```

Comment:
Consider providing a docstring here.  If you do, it would be worth mentioning the requirements of `ctx`.

Reply:
Done